tests/integration/istioio/trafficmanagement/scripts/request_timeouts.txt

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Export HOST, PORT and URL
+# $snippet export_gateway_url_1.sh syntax="bash"
+$ export INGRESS_HOST={{ .ingressHostCommand }}
+$ export INGRESS_PORT={{ .ingressPortCommand }}
+$ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+# $endsnippet
+
+# Step 1: Route requests to v2 of the reviews service
+# $snippet v2_reviews.sh syntax="bash"
+$ kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  hosts:
+    - reviews
+  http:
+  - route:
+    - destination:
+        host: reviews
+        subset: v2
+EOF
+# $endsnippet
+
+# Step 2: Add a 2 second delay to calls to the ratings service
+# $snippet delay_ratings.sh syntax="bash"
+$ kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+  - ratings
+  http:
+  - fault:
+      delay:
+        percent: 100
+        fixedDelay: 2s
+    route:
+    - destination:
+        host: ratings
+        subset: v1
+EOF
+# $endsnippet
+
+echo "sending request to $GATEWAY_URL"
```

Comment:
you will likely need an `istioctl experimental wait` here to wait for the virtual service updates.

Reply:
Added 
```go
istioctl experimental wait --for=distribution VirtualService ratings.default --timeout=60s
```
But keep getting following error, even when I increase `--timeout` to `120s` or `180s`.
```go
Error: timeout expired before resource virtual-service/default/ratings became effective on all sidecars
```
I believe `istioctl wait` is not considering custom `--timeout`. 

Any thoughts ?